components/atoms/button/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+import styles from './styles.css'
+
+const Button = ({ onClick, disabled, className, children }) => (
+  <button
+    className={classNames(styles.button, className, { [styles.disabled]: disabled })}
+    onClick={disabled ? null : onClick}
```

Comment:
you don't need it. When the button is `disabled` the browser automatically won't run the onClick function.